src/commonMain/kotlin/spaceEngineers/model/SeBlock.kt

```diff
@@ -1,14 +1,15 @@
 package spaceEngineers.model
 
 
-class SeBlock(
+data class SeBlock(
     override val id: String,
     override val position: Vec3,
+    val blockType: String
 ) : SeEntity {
     var maxIntegrity = 0f
     var buildIntegrity = 0f
     var integrity = 0f
-    var blockType: String? = null
+
     var minPosition: Vec3? = null
```

Comment:
Why this change?

Reply:
We don't necessarily need this and we can always revert. I did it because data class has nice pre-generated toString representation and I use it for debugging.

Reply:
OK